cupy_backends/cuda/api/runtime.pyx

```diff
@@ -909,6 +915,10 @@ cpdef streamAddCallback(intptr_t stream, callback, intptr_t arg,
 
 
 cpdef launchHostFunc(intptr_t stream, callback, intptr_t arg):
+    # N.B. Currently this function should not be called during CUDA graph
+    # capture, as the reference to the callback/arg will be DECREFed on the
+    # first callback.
+    # Eventually this should be replaced by `_launchHostFuncUnmanaged`.
```

Comment:
This PR keeps `cupy.cuda.runtime.launchHostFunc` backward-compatible as the fix needs to be backported to v13. Maybe we can change this in v14 in a follow-up PR.